apps/web-staking/src/app/components/announcementBanner/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React, {useEffect, useState} from 'react';
+import ExternalLinkIcon from "../../../../../../packages/ui/src/rebrand/icons/ExternalLinkIcon";
+import {CloseIcon} from "../../../../../../packages/ui/src/rebrand/icons/CloseIcon";
+
```

Comment:
Component file name should not be index.tsx.
Component should be included in the file and function name.